index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function (a, b) {
+      return a - b
+    });
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < this.length) {
+      return this.items[pos];
+    } else {
+      throw new Error('OutOfBounds');
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error('EmptySortedList');
+    } else {
+      return Math.max(...this.items)
+    }
+  }
 
-  min() {}
+  min() {
```

Comment:
```
  min() {
    // if the array is empty (!this.length)
    if (this.items.length === 0) {
      throw new Error('EmptySortedList');
    }

    return Math.min(...this.items);

    // or, since we work with sorted array, we can simply point the first element of
    // the array as the lowest value of the array
    // return this.items[0];
  }

```